app/resources/v1/application_resource.rb

```diff
@@ -79,9 +77,11 @@ def current_user
     context.fetch(:user)
   end
 
+  # :nocov:
   def current_application
```

Comment:
This method seems unused

Reply:
this is an endpoint, right? I suspect that's why it isn't covered by tests (or do we test endpoints too?) Did you check whether amber-ui depends on it?